docs/compression_algorithms/Pruning.md

```diff
@@ -82,15 +94,25 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
     "params": {
         "schedule": "baseline", // The type of scheduling to use for adjusting the target pruning level. Either `exponential`, `exponential_with_bias`,  or `baseline`, by default it is `baseline`"
         "pruning_target": 0.4, // Target value of the pruning level for the convolutions that can be pruned. These convolutions are determined by the model architecture. 0.5 by default.
+        "pruning_flops_target": 0.4, // Target value of the pruning level by FLOPs in the whole model. Only one parameter from `pruning_target` and `pruning_flops_target` can be set. If if none of them are specified, `pruning_target` = 0.5 will be default value. 
```

Comment:
If if => If
are specified => is specified
`pruning_target` = 0.5 will be default value. => `pruning_target` = 0.5 is used as the default value.

Reply:
Done